shared/src/components/exercise/index.cjsx

```diff
@@ -104,7 +104,7 @@ ExerciseMixin =
     {parts} = @props
     step = _.last(parts)
 
-    related_content = step.related_content if related
+    related_content = step.related_content # if related
 
```

Comment:
coo, in that case can we remove the `#` :)  could also be: `{ related_content } = step;` but that's no biggy